scudo-whitepaper.md

```diff
@@ -163,7 +163,9 @@ Once the Director repository decides which images are required and signs targets
 
 After all the images and metadata from both repositories have been delivered to the automobile’s primary ECU, it can perform a modified version of Uptane’s verification workflow. Consider section [5.4.4.2 of the *Uptane Standard*](https://uptane.github.io/papers/uptane-standard.2.0.0.html). At step 10, after verifying that the Targets metadata of Director and Image repositories match for an image, the in-toto metadata for the image, the root layout, and corresponding public keys are fetched. These must, of course, also match what’s signed in the Uptane/TUF metadata at the Image repository. Then, primary and full verification ECUs may execute the in-toto verification workflow to verify the supply chain of the image.
 
-Performing full in-toto verification can be a resource intensive process for some vehicle ECUs. As a stopgap measure, the Image repository may perform in-toto verification to generate a new “summary link” metadata file. The link can be used as proof that the Image repository performed the complete verification workflow, and the repository can then send this summary link metadata to the vehicle. Using this lightweight verification workflow requires more trust in the Image repository, as it is performing the verification rather than the client, which in this case would be the primary ECU. Further, in these situations, the Image repository could become a single point of failure for thousands of vehicles. So, while this approach can significantly reduce the metadata and verification overhead, it must only be used when full verification is categorically impossible on the vehicle.
+Performing full in-toto verification can be a resource intensive process for some vehicle ECUs, such as those designated as partial verification ECUs in Uptane. In these situations, other ECUs capable of performing in-toto verification are expected to step in for the the ECUs that cannot verify for themselves. This responsibility could fall to the primary ECU or a collection of full verification secondaries that share the extra load on behalf of partial verification secondary ECUs.
+
+For vehicles with no ECUs capable of performing in-toto verification, as a stopgap measure, the Image repository may perform in-toto verification to generate a new “summary link” metadata file. The link can be used as proof that the Image repository performed the complete verification workflow, and the repository can then send this summary link metadata to the vehicle. Using this lightweight verification workflow requires more trust in the Image repository, as it is performing the verification rather than the client, which in this case would be the primary ECU. Further, in these situations, the Image repository could become a single point of failure for thousands of vehicles. So, while this approach can significantly reduce the metadata and verification overhead, it must only be used when full verification is categorically impossible on the vehicle.
```

Comment:
Move "as a stopgap measure" to the front of the first sentence --- "As a stopgap measure for vehicles with no ECUs capable of performing in-toto verification, the Image repository may..." Also again Primary ECU